supersetapiclient/base.py

```diff
@@ -25,6 +25,7 @@ class Object:
     _parent = None
     EXPORTABLE = False
     JSON_FIELDS = []
+    JSON_AS_OBJECT_FIELDS = {}
```

Comment:
Should be called `LIST_OF_OBJECT_FIELDS`? Or rename all `LIST_OF_OBJECT_FIELDS` to `JSON_AS_OBJECT_FIELDS` which is probably a better name.

Reply:
Indeed it should. I guess I did end up with a wrong name change. I relied on the IDE to make the change.
```suggestion
    LIST_OF_OBJECT_FIELDS = {}
```